DataConnectors/SentinelOne/SentinelOneSentinelConnector/__init__.py

```diff
@@ -160,9 +218,21 @@ def reports_list(self):
                 next_page_token = self.get_threat_data("/web/api/v2.1/threats/", "Events", id,
                                                   params = api_req_params)
 
+        #gets installed applications for given agent computer name
+        for name in self.computerName_arr:
+            logging.debug("Getting report: installed apps")
+            next_page_token = self.get_installed_apps("/web/api/v2.1/installed-applications", "Installed-apps", name)
+            while next_page_token:
+                api_req_params = {"cursor": next_page_token}

```

Comment:
Shouldn't we be sending api_req_params as parameter to get_installed_apps() function. It may not read the next page.

Reply:
yes correct, will make that change.